spinalcordtoolbox/qmri/mt.py

```diff
@@ -44,8 +44,13 @@ def compute_mtr(nii_mt1, nii_mt0, threshold_mtr=100):
     :param threshold_mtr: float: value above which number will be clipped
     :return: nii_mtr
     """
+    # Convert input before compute MTR
```

Comment:
Nitpick, but we might want to use a more descriptive comment here... right now, I think this comment isn't adding much?

```suggestion
    # Convert input to avoid numerical errors from int16 data (https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3636)
```